docs/development-setup.md

```diff
@@ -1,73 +1,137 @@
 # Local Development
 
-It's actually very simple to setup OpenShift locally. We'll use a docker image from Docker Hub. For more OpenShift-focused information, see [this guide](https://docs.openshift.org/latest/getting_started/administrators.html).
 
-First thing to do is to clone this git repo:
+## Running OpenShift
+
+For more options how to install and run OpenShift, please see [this page](https://install.openshift.com/).
+
+We'll use the `oc cluster up` method, since it's very easy to use:
+
+ 1. If you're on Fedora, you can install package origin:
+    ```
+    dnf install -y origin
+    ```
+    Now you have command `oc` available on your system
+
+ 2. You can start a single-node cluster like this
+    ```
+    oc cluster up
+    ```
+    (If you are running into some connection issues, please flush your iptables
+    rules bufore starting the cluster with `iptables -F`)
+
+
+## Getting `osbs-client`
+
+For local development, we advise to clone this git repo:
 
 ```
 $ git clone git@github.com:projectatomic/osbs-client.git
 $ cd osbs-client
 ```
 
+Or if you just want to try it out, you can install it from Fedora repositories:
+
+```
+$ dnf install -y osbs-client
+```
 
-## setup with docker-compose
 
-Install `docker-compose`:
+## Obtaining build image
+
+In order to build *your* images, you need to have a `buildroot` image first.
+This image is used to create a build container, where `atomic-reactor` is
+running and taking care of building your images.
+
+If you cloned this repository, you can use `Dockerfile` which is present in root directory.
 
 ```
-$ dnf install -y docker-compose
+$ docker build --no-cache --tag=buildroot .
 ```
 
-Then you can start OpenShift with one simple command:
+If you are using `osbs-client` from git master branch, you should also install
+git master of atomic-reactor:
 
 ```
-$ docker-compose up -d
+$ docker build --no-cache --tag=buildroot --build-arg LATEST_CODE=true .
```

Comment:
Shouldn't this have `REACTOR_SOURCE=git` instead of `LATEST_CODE=true`?

For those who are not masters of docker yet, would be nice to include who this step can be verified (`docker images` and look for *buildroot*) - error messages are not always obvious and can be easily missed.